ddp-workspace/projects/ddp-dsm-ui/src/app/scanner/scanner.component.ts

```diff
@@ -35,6 +35,8 @@ export class ScannerComponent implements DoCheck, OnDestroy {
   });
   public additionalMessage: string | undefined;
 
+  private shortIds = [];
```

Comment:
This field is accessed from the template, therefore it has to be public. While JIT is okay with that, AOT might have some problems.